2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_setup.py:_flush():76] Current SDK version is 0.15.2
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_setup.py:_flush():76] Configure stats pid to 3334853
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_setup.py:_flush():76] Loading settings from /home/maolee/.config/wandb/settings
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_setup.py:_flush():76] Loading settings from /nfs/turbo/isr-fconrad1/projects/corpus-variability/src/wandb/settings
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_init.py:_log_setup():507] Logging user logs to /nfs/turbo/isr-fconrad1/projects/corpus-variability/src/wandb/run-20240620_130101-v1ig33bi/logs/debug.log
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_init.py:_log_setup():508] Logging internal logs to /nfs/turbo/isr-fconrad1/projects/corpus-variability/src/wandb/run-20240620_130101-v1ig33bi/logs/debug-internal.log
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_init.py:init():547] calling init triggers
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2024-06-20 13:01:01,639 INFO    MainThread:3334853 [wandb_init.py:init():571] re-initializing run, found existing run on stack: ta45av9r
2024-06-20 13:01:01,640 INFO    MainThread:3334853 [wandb_run.py:_finish():1893] finishing run maoli/sft_embedding_decoding/ta45av9r
2024-06-20 13:01:01,641 INFO    MainThread:3334853 [jupyter.py:save_history():445] not saving jupyter history
2024-06-20 13:01:01,641 INFO    MainThread:3334853 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-06-20 13:01:01,641 INFO    MainThread:3334853 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2024-06-20 13:01:01,641 INFO    MainThread:3334853 [wandb_run.py:_atexit_cleanup():2127] got exitcode: 0
2024-06-20 13:01:01,641 INFO    MainThread:3334853 [wandb_run.py:_restore():2110] restore
2024-06-20 13:01:01,641 INFO    MainThread:3334853 [wandb_run.py:_restore():2116] restore done
2024-06-20 13:01:04,072 INFO    MainThread:3334853 [wandb_run.py:_footer_history_summary_info():3469] rendering history
2024-06-20 13:01:04,072 INFO    MainThread:3334853 [wandb_run.py:_footer_history_summary_info():3501] rendering summary
2024-06-20 13:01:04,076 INFO    MainThread:3334853 [wandb_run.py:_footer_sync_info():3428] logging synced files
2024-06-20 13:01:04,180 INFO    MainThread:3334853 [wandb_init.py:init():596] starting backend
2024-06-20 13:01:04,180 INFO    MainThread:3334853 [wandb_init.py:init():600] setting up manager
2024-06-20 13:01:04,181 INFO    MainThread:3334853 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-06-20 13:01:04,182 INFO    MainThread:3334853 [wandb_init.py:init():606] backend started and connected
2024-06-20 13:01:04,186 INFO    MainThread:3334853 [wandb_run.py:_label_probe_notebook():1238] probe notebook
2024-06-20 13:01:04,186 INFO    MainThread:3334853 [wandb_run.py:_label_probe_notebook():1248] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-06-20 13:01:04,186 INFO    MainThread:3334853 [wandb_init.py:init():700] updated telemetry
2024-06-20 13:01:04,219 INFO    MainThread:3334853 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2024-06-20 13:01:05,297 INFO    MainThread:3334853 [wandb_run.py:_on_init():2177] communicating current version
2024-06-20 13:01:05,379 INFO    MainThread:3334853 [wandb_run.py:_on_init():2186] got version response upgrade_message: "wandb version 0.17.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-06-20 13:01:05,379 INFO    MainThread:3334853 [wandb_init.py:init():787] starting run threads in backend
2024-06-20 13:01:09,381 WARNING MainThread:3334853 [wandb_init.py:init():1172] interrupted
Traceback (most recent call last):
  File "/home/maolee/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1150, in init
    run = wi.init()
  File "/home/maolee/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 799, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/home/maolee/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/maolee/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/maolee/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/sw/pkgs/arc/python3.10-anaconda/2023.03/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/sw/pkgs/arc/python3.10-anaconda/2023.03/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
